streaming/src/main/scala/org/apache/carbondata/streaming/StreamHandoffRDD.scala

```diff
@@ -99,6 +101,38 @@ class StreamHandoffRDD[K, V](
     formatter.format(new Date())
   }
 
+  private def setTempDirLocation(split: Partition) : Unit = {
```

Comment:
Better to reuse this code for CarbonMergerRDD , SparkPartitionLoader , AlterTableLoadPartitionRDD,  DataLoadProcessorStepOnSpark and CarbonFileFormat

Reply:
SparkPartitionLoader, DataLoadProcessorStepOnSpark, CarbonFileFormat uses array of locations to speed up the loading, can't use the same function to achieve the functionality